mlir/include/mlir/Dialect/MIGraphX/MIGraphXOps.td

```diff
@@ -160,30 +160,32 @@ def MIGraphX_FloorOp :
 
 def MIGraphX_QuantizeLinearOp :
     MIGraphX_Op<"quantizelinear">,
-    Arguments<(ins TensorOf<[F32]>:$input,
+    Arguments<(ins TensorOf<[F32, F16]>:$input,
                    TensorOf<[F32]>:$scale,
-                   Optional<TensorOf<[I32]>>:$bias)>,
-	  Results<(outs TensorOf<[I8]>:$output)> {
+                   Optional<TensorOf<[AnyInteger]>>:$bias)>,
+	  Results<(outs TensorOf<[AnyInteger]>:$output)> {
```

Comment:
I'll note that the API here needs to exactly match MIGraphX's IR

Reply:
... wait, hold on, are these changes actually part of your PR or is the diff just noisy?